core-developers/memorialization.rst

```diff
@@ -0,0 +1,146 @@
+content of their accounts without changes after they've passed away.
+To support the bereaved, in some instances, we may remove or change
+certain content when the legacy contact or family members request it.
+
+GitHub
+------
+
+* The user is removed from the `python/ <https://github.com/orgs/python/>`_
+  organization on GitHub;
+* The user is removed from the `psf/ <https://github.com/orgs/psf/>`_
+  organization on GitHub;
+* The user is removed from the `pypa/ <https://github.com/orgs/pypa/>`_
+  organization on GitHub.
+
+The PSF staff does not follow up with GitHub with regards to GitHub account
+cancellation as this action is reserved for next-of-kin or designated by
+the deceased GitHub user to act as an account successor.
+
+The general policy regarding deceased users on GitHub is described
+`here <https://docs.github.com/en/site-policy/other-site-policies/github-deceased-user-policy>`_.
+
+Repositories in the organization
+--------------------------------
+
+* The user's GitHub handle is removed from ``/.github/CODEOWNERS``.
+  To see all that need action, perform
+  `this query <https://github.com/search?q=org%3Apython+path%3A**%2F.github%2FCODEOWNERS+USERNAME&type=code>`_.
+* The user is marked as deceased in the private
+  `voters/python-core.toml <https://github.com/python/voters/blob/main/python-core.toml>`_
+  file with the ``left=`` field set to the day of passing, if known.
+
+discuss.python.org
+------------------
+
+* The user's "custom status" is set to 🕊 ``in memoriam``;
+* The user's "about me" is amended with ``$firstname passed away on $date. [In memoriam.]($in_memoriam_post_url)``;
+* In the user's security "recently used devices" the staff member chooses "Log out all";
+* In the user's permissions the staff member chooses "Deactivate account";
+* The user's trust level is reset to ``1: basic user`` (trust level 0 doesn't allow links in "About Me");
+* The user's "associated accounts" (like GitHub) that provide an alternative
+  login method, are all disconnected;
+* The user's API keys are revoked;
+* The user's admin or moderator right is revoked;
+* The user's primary email address is reset to ``USERNAME@in-memoriam.invalid`` and
+  secondary email addresses are removed (this step requires the administrator
+  to contact Discourse.org staff via ``team@discourse.org``)
+
+The "in memoriam" Discourse topic mentioned above is best created by
+a community member close to the deceased.
+
+The general best practice for deceased community members on
+Discourse-powered forums is described `here <https://meta.discourse.org/t/best-practices-for-deceased-community-members/146210>`_.
+
+python.org email account
+------------------------
+
+The PSF staff member emails ``postmaster@python.org`` to ask the email
+administrator to:
+
+* remove SMTP access from ``USERNAME@python.org``;
+* reset the password to POP3/IMAP for ``USERNAME@python.org``;
+* disable email forwarding, if set up, for ``USERNAME@python.org``;
```

Comment:
@ambv - apologies to being late in the game on this, but only noticed this PR this morning.
Removing email forwarding like this may cause problems for the deceased's family or estate executors:
- if folks are only communicating with the deceased through their `python.org` address, their family or estate executors may have a harder time if this suddenly stops working
- confusion may crop up if the address ends up being re-used; I've hard problems in the past where various email addresses and social media handles have ended up being reused after a family member passed away.

Reply:
I'll add a note to avoid name reuse forever.

As for keeping the email address alive forever, I'm afraid this is out of the question due to security concerns. Removing the ability to send email (SMTP) and receive email (POP3/IMAP/forwarding) via the PSF-provided address after the user passed away is done to disallow password resets and other forms of impersonation.

Per the [python.org email policy](https://www.python.org/psf/records/board/policies/email/), that email address is provided on a best-effort basis, which suggests it should not be used by its users as a primary communication channel.

I will add a note that in case the email shutdown causes issues for the estate executors, the PSF will reasonably try to help if contacted directly.

Reply:
Addressed in #1517.

Reply:
Totally understand the SMTP and POP3/IMAP side of things, although for the latter, if it's easy enough to do, might be helpful to send an archive to family/executors.

If an address is just forwarding, what's the harm? I appreciate it's best efforts basis :-)